templates/webmention-comment-form.php

```diff
@@ -1,13 +1,31 @@
+<?php
+/**
+ * Hook to add custom content before the webmention form added to the comment form.
+ */
+do_action( 'webmention_comment_form_template_start' );
+?>
 <form id="webmention-form" action="<?php echo get_webmention_endpoint(); ?>" method="post">
 	<p>
 		<label for="webmention-source"><?php echo get_webmention_form_text( get_the_ID() ); ?></label>
 	</p>
 	<p>
-		<input id="webmention-source" type="url" autocomplete="url" name="source" placeholder="<?php esc_attr_e( 'URL/Permalink of your article', 'webmention' ); ?>" />
+		<input
+			id="webmention-source"
+			type="url"
+			autocomplete="url"
+			name="source"
+			placeholder="<?php esc_attr_e( 'URL/Permalink of your article', 'webmention' ); ?>"
+		/>
```

Comment:
I only updated this to make sonarcloud happier (it was reporting a fail because the line was too long).

Reply:
I have to remove sonarcube, it is way to pedantic :)

Reply:
please ignore it by now, I will switch to PHPCS!

Reply:
👍 I've removed that clean-up commit.